### DISCUSSION TITLE
Reach out to Elixir community about the current state of Gleam <> Elixir interop

### DISCUSSION THREAD

**Discussion by michallepicki** [gleam-lang/gleam#2289] [Ideas & suggestions]
There has been good progress on using Elixir from Gleam in recent months. From my experience many Elixir folks are pretty positive about Gleam, but not everyone in the Elixir community might be aware of that progress.

There is also still no support for the Gleam build tool from Elixir (Mix) side.

I think it would be good to reach out to the Elixir community (maybe on Elixir Forum?) to give an update on the current state of interop. To make it easier to understand, we could give examples of possibilities and limitations. And I think it's fine to ask for help on figuring out Gleam support in Mix - maybe some volunteers would show up.

**Comment by lpil** [comment]
Fab idea, thank you
